### ISSUE TITLE
Does model also provides each landmark confidence score

### ISSUE THREAD

**Issue by sainisanjay** [github-luffy/PFLD_68points_Pytorch#47]
@github-luffy Does models also provides each landmark confidence score? if no then have any idea how we can get from network?

**Comment by github-luffy** [comment]
PFLD uses the regression loss function, and the cross entropy loss of the classification is not used, and there is no confidence score.

**Comment by sainisanjay** [comment]
@github-luffy  Thanks for your response, may i know what modification i can do in n/w in order to get score of each landmarks?

**Comment by github-luffy** [comment]
To output the key point heat map, you can look at the papers and projects of cpm (convolutional pose machine)
